### PR TITLE
statichomepage

### PR DESCRIPTION
**Pull request by preethasri**
created statichome page with sidebar,feedbar and aside
Screenshot (37)

### CODE REVIEW

File: src/Components/Aside/Aside.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                        <div className="aside-item-btn">
+                            <button className="follow-btn">Follow
+
+                            </button>
+
+                        </div>
+
+                    </div>
+                    </div>
+                    {/*  */}
```
Comment (Chakravarthi0) [inline src/Components/Aside/Aside.js:34]: you have this kind of empty comment in several places, you can remove these

**Review by Chakravarthi0 — APPROVED** [review]
suggestion: You can use rem units instead of px.
everything else looks good

**Review by manojasarsa — APPROVED** [review]
Looks good! GTG!

File: src/Components/Aside/Aside.js (hunk 1 above)
Comment (preethasri) [reply src/Components/Aside/Aside.js:34]: sure saran will work on this

**Pull request merged**